README.md

```diff
@@ -36,6 +40,42 @@ The idea of these resources is, to offer similar functionality as the
 - `/saul/dev` (POST) needs an ID as argument. Returns some information
   about the sensor for that ID (name and type). 
 
+## Phydat as CBOR
```

Comment:
Please add a short explanation what CBOR is.

Reply:
Please see 643319649963a74d569d6dd6ed96f18e6b287ec4; do you think this is enough?